.pyup.yml

```diff
@@ -0,0 +1,4 @@
+# autogenerated pyup.io config file 
+# see https://pyup.io/docs/configuration/ for all available options
+
+update: insecure
```

Comment:
I just experimented with a private repo and `insecure` works perfectly fine. The shield also uses the term `insecure`:

[![Updates](https://pyup.io/repos/github/gratipay/inside.gratipay.com/shield.svg)](https://pyup.io/repos/github/gratipay/inside.gratipay.com/)

My guess is that they renamed it and forgot to update the docs.